### ISSUE TITLE
Problem with version v0.10.2 (p5js)

### ISSUE THREAD

**Issue by rodrigoKulb** [emilyxxie/green_rain#19]
Problem with version v0.10.2 (p5js)

First congratulations on the script, I was testing with version 0.10.2 and I found a problem with the word symbol, I believe it is a reserved word in this version.

I switched to symbolM and it worked!

**Comment by czkoudy** [comment]
Yes, the ES6 comes with Symbol - [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol)

So, as you mentioned, using any other name will work :-)

**Comment by PythonCoderUnicorn** [comment]
Hello

I copied the code from p5js file and pasted it into my p5js browser editor and it kept giving me errors. What fixed the error was to renames `Symbol()` function to something else, in my case `Action()` in both places it was called. 

Thank you so much for the code, I always wondered how this was created.
